### PR TITLE
[#199] default to WildFly connection configuration

### PR DESCRIPTION
**Pull request by simkam**
fixes https://github.com/wildfly-extras/creaper/issues/199

- switch default protocol to `http-remoting`
- switch default port to 9990
- drop `creaper.wildfly` property
- drop `remoting` protocol support

### CODE REVIEW

File: core/src/main/java/org/wildfly/extras/creaper/core/online/OnlineOptions.java [hunk 1]
```diff
@@ -196,10 +191,8 @@ private ConnectionOnlineOptions(Data data) {
 
         /**
          * <p>Connect to {@code localhost} and use the default management port of the application server.
-         * This is {@code 9999} by default (JBoss AS 7), and if the system property {@code creaper.wildfly} is defined,
-         * it changes to 9990 or (if {@link OptionalOnlineOptions#ssl(SslOptions) ssl} is used) to 9993 (WildFly).
-         * This makes it easy to run the same code against both AS7 and WildFly only by defining a single system
-         * property.</p>
+         * This is {@code 9990} by default, but if {@link OptionalOnlineOptions#ssl(SslOptions) ssl} it is 9993.
```
Comment (jstourac) [inline core/src/main/java/org/wildfly/extras/creaper/core/online/OnlineOptions.java:194]:
```suggestion
         * This is {@code 9990} by default, but if {@link OptionalOnlineOptions#ssl(SslOptions) ssl} is not null, then it is 9993.
```

File: testsuite/pom.xml [hunk 2]
```diff
@@ -1125,7 +1123,7 @@
                 <arquillianContainer.version>${wildfly25.arquillianContainer.version}</arquillianContainer.version>
 
                 <tests.ignoredCategory>${ignoredCategory.wildfly11}</tests.ignoredCategory>
-                <tests.specialJvmArgs>${specialJvmArgs.wildfly}</tests.specialJvmArgs>
+                <tests.specialJvmArgs></tests.specialJvmArgs>
```
Comment (jstourac) [inline testsuite/pom.xml:1126]: well, we could probably delete these all `specialJvmArgs` properties but whatever...

**Comment by jstourac** [comment]
LGTM, thank you - just one actual small concern regarding one comment line.

File: testsuite/pom.xml (hunk 2 above)
Comment (simkam) [reply testsuite/pom.xml:1126]: let's keep it for now, if we ever need it. It might be useful for JDK17 for example.

**Pull request merged**